atd-vzd/migrations/default/1715960023005_socrata_export_views/up.sql

```diff
@@ -92,7 +92,7 @@ where
 create or replace view socrata_export_people_view as (
     select
         people.id as id,
-        people.id as person_id, --legacy support
+        people.id as person_id, --this column exists for backwards compatibility
```

Comment:
wait, why do we have two person id columns?

Reply:
only because i feel bad "breaking" the public dataset if folks have been using the `person_id` column. maybe that's not worth it though 🤷

Reply:
ahhh i see. i dont have a heavy opinion on this :shrug: